domino/flyte/task.py

```diff
@@ -81,6 +82,11 @@ class DominoJobConfig(object):
     ComputeClusterProperties: Optional[ClusterProperties] = None
     VolumeSizeGiB: Optional[float] = None
     ExternalVolumeMountIds: Optional[List[str]] = None
+    # we want to provide the following to the flyte agent, but these are set by the DominoJobTask constructor 
+    # and so would never be expected to be passed into a DominoJobConfig constructor by a user.
+    # they are included here for completeness documenting what task config values are sent to/read by the flyte agent.
+    _InputInterfaceBase64: Optional[str] = None
```

Comment:
@noahjax this part is a bit odd, but including them here seemed better than leaving out -- what do you think?

Reply:
I'd leave them out, I don't think the user needs to know or think about these

Reply:
ok, i'll take them out